From 99d654438cb259bee70d23363b2d5aefee830c7e Mon Sep 17 00:00:00 2001
From: Jim Fehlig <jfehlig@suse.com>
Date: Tue, 5 Jul 2022 11:56:28 -0600
Subject: apparmor: Add rule allowing qemu to write screenshots

Not sure who added this patch. See bug#904426 for possible suspects.
---
 src/security/apparmor/libvirt-qemu | 3 +++
 1 file changed, 3 insertions(+)

Index: libvirt-8.7.0/src/security/apparmor/libvirt-qemu
===================================================================
--- libvirt-8.7.0.orig/src/security/apparmor/libvirt-qemu
+++ libvirt-8.7.0/src/security/apparmor/libvirt-qemu
@@ -253,3 +253,6 @@
   # required for QEMU accessing UEFI nvram variables
   owner /var/lib/libvirt/qemu/nvram/*_VARS.fd rwk,
   owner /var/lib/libvirt/qemu/nvram/*_VARS.ms.fd rwk,
+
+  # Temporary screendump rule -- See bsc#904426
+  /var/cache/libvirt/qemu/qemu.screendump.* rw,
